specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/examples/ClassifyImageUrlWithNoStore.json

```diff
@@ -0,0 +1,35 @@
+{
+    "parameters": {
+        "Endpoint": "https://westus.api.cognitive.microsoft.com",
+        "Content-Type": "application/json",
+        "Prediction-Key": "{API Key}",
+        "projectId": "64b822c5-8082-4b36-a426-27225f4aa18c",
+        "publishedModelName": "MyModel1",
+        "imageUrl": {
+            "url": "{Image URL}"
+        }
+    },
+    "responses": {
+        "200": {
+            "headers": {},
+            "body": {
+                "id": "951098b2-9b69-427b-bddb-d5cb618874e3",
+                "project": "64b822c5-8082-4b36-a426-27225f4aa18c",
+                "iteration": "fe1e83c4-6f50-4899-9544-6bb08cf0e15a",
+                "created": "2017-12-19T14:21:41.6789561Z",
```

Comment:
Unless specifically required, please limit precision to milliseconds (ECMA script's subset of ISO8601).

Reply:
Will investigate

Reply:
Updated backend and examples to limit precision